T.A.Password Strength/index.js

```diff
@@ -0,0 +1,26 @@
+const pass = document.getElementById('password');
+const msg = document.getElementById('message');
+const str = document.getElementById('strength');
+
+pass.addEventListener('input', () => {
+    if(pass.value.length > 0) {
```

Comment:
Add logic to check for password strength as,
- At least 6 characters
- Must have one uppercase letter
- Must have one number
- Must have a special character.
- Must Not have a space.

Reply:
https://thirsty-meninsky-e60125.netlify.app